### ISSUE TITLE
[NCPA 2.0.1/Windows Update] Restart of services results in: Error 1053

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#294]
Restarting both services: ncpalistener, ncpapassive takes very long and results in: Error 1053. The state of both services, after this error message is: Will be terminated. Both processes are running. You have to kill the processes via task manager and then it is possible to start both services.

**Comment by ccztux** [comment]
Content of service logfile ```win32service_ncpalistener.log```:
```
[04284] 2017/01/04 10:07:10.245 starting logging at level ERROR
[01456] 2017/01/04 10:07:25.832 Python exception encountered:
[01456] 2017/01/04 10:07:25.832     Internal Message: exception calling Stop method
[01456] 2017/01/04 10:07:25.832     Type => <type 'exceptions.AttributeError'>
[01456] 2017/01/04 10:07:25.832     Value => '_Event' object has no attribute 'Set'
[01456] 2017/01/04 10:07:25.832     Traceback (most recent call last):

[01456] 2017/01/04 10:07:25.832       File "C:\ncpa\agent\ncpa_windows.py", line 112, in Stop

[01456] 2017/01/04 10:07:25.832     AttributeError: '_Event' object has no attribute 'Set'
```

OS Version: ```Windows 7 SP1```

**Comment by jomann09** [comment]
Fixed in the re-built version of 2.0.1 since it's just a windows issue... this was a strange issue with the fact that normal threading (shown to work by the writer of cx_freeze for services) does not actually work with the win32service base ... the current .exe for 2.0.1 should not have this issue. You will need to manually stop ncpa_listener.exe and ncpa_passive.exe though before installing the new version.
